pkg/cmd/extension/command.go

```diff
@@ -141,9 +141,43 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "create <name>",
+			Short: "Create a new extension",
```

Comment:
Just had another thought: should this say `Create a new extension locally` so that it's clear that whatever will be created, it's only on your machine?

It can be either here or in the `Long` description. Or maybe it's not necessary. Just an idea!